Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    truemail (0.1.10)
+    truemail (0.1.11)
```

Comment:
I think you should increment the second number because of this feature, rather than bugfix

Reply:
Yup :+1: Fixed.